new-contract-interaction/src/agent.js

```diff
@@ -0,0 +1,218 @@
+const {
+  Finding, FindingSeverity, FindingType, getEthersProvider,
+} = require('forta-agent');
+
+const config = require('../agent-config.json');
+
+// set up a variable to hold initialization data used in the handler
+const initializeData = {};
+
+// helper function to create alert for contract interaction
+function createContractInteractionAlert(
+  contractName,
+  contractAddress,
+  interactionAddress,
+  blockNumber,
+  findingType,
+  findingSeverity,
+  protocolName,
+  protocolAbbreviation,
+  developerAbbreviation,
+) {
+  const finding = {
+    name: `${protocolName} New Contract Interaction`,
+    description: `The ${contractName} contract interacted with a new contract ${interactionAddress}`,
+    alertId: `${developerAbbreviation}-${protocolAbbreviation}-NEW-CONTRACT-INTERACTION`,
+    type: FindingType[findingType],
+    severity: FindingSeverity[findingSeverity],
+    protocol: protocolName,
+    metadata: {
+      contractName,
+      contractAddress,
+      interactionAddress,
+      blockNumber,
```

Comment:
Transaction and block information is provided by forta explorer so we don't need to add `blockNumber` to the metadata